### ISSUE TITLE
~~Could you offer an darwin_arm64 package?~~ Update Readme

### ISSUE THREAD

**Issue by h0jeZvgoxFepBQ2C** [exoscale/terraform-provider-exoscale#187]
Hi,

it seems that I cannot use Exoscale/Terraform on my new Macbook:

```
│ Error: Incompatible provider version
│
│ Provider registry.terraform.io/exoscale/exoscale v0.18.2 does not have a package available for your current platform, darwin_arm64.
│
│ Provider releases are separate from Terraform CLI releases, so not all providers are available for all platforms. Other versions of this provider may have different platforms supported.
╵
```

Could you provide a package?

**Comment by h0jeZvgoxFepBQ2C** [comment]
Ah, it seems that the newest version of your provider works on the new OSX architecture.

Could you update your README?
https://registry.terraform.io/providers/exoscale/exoscale/latest/docs

It specifies the 0.18.2 version.

**Comment by kobajagi** [comment]
Thank you for feedback. As stated we do support `darwin_arm64` architecture. We will update our docs in the next release to reference new provider version.

**Comment by PhilippeChepy** [comment]
Resolved in v0.40.0 (soon available in the terraform registry).
